### PR TITLE
relicense to GPL3

### PR DESCRIPTION
**Pull request by majestrate**
this officially makes lokinet GPL3

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -1,23 +1,683 @@
-LokiNET is the reference implementation of LLARP (Low Latency Anonymous 
-Routing Protocol) and is licensed under ZLIB license, however the protocol 
-specifications are placed into the public domian using the CC0 License.
+LokiNET is the reference implementation of LLARP (Low Latency Anonymous
+Routing Protocol); this distribution of LokiNET and the Lokinet control
+panel is licensed under the GNU General Public License, version 3.
```
Comment (jagerman) [inline LICENSE.txt:3]: You don't want this control panel stuff; that was there just for the control panel itself.  Instead:

`is the reference implementation of LLARP (Low Latency Anonymous
Routing Protocol).  LokiNET is licensed under the GNU General Public License, version 3.`

Comment (jagerman) [reply LICENSE.txt:3]: Actually just using the GNU-recommended preamble might be better:

    LokiNET is the reference implementation of LLARP (Low Latency Anonymous 
Routing Protocol).

    This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.

(Do we want the "or later" part?)

**Comment by majestrate** [comment]
something to think about, LGPL3 allows linking to non gpl code, may want to consider that instead of flat gpl3

**Comment by jagerman** [comment]
> something to think about, LGPL3 allows linking to non gpl code, may want to consider that instead of flat gpl3

For lokinet as it currently exists that won't make any difference.  We probably *do* want to separate the full application from the core logic though, though, and when we do so we can consider whether to license the library parts as LGPL.

**Pull request merged**